Testscripts/Windows/CLEANUP-DiffDiskGrowth.ps1

```diff
@@ -202,4 +203,4 @@ function Main {
 }
 
 Main -VMName $AllVMData.RoleName -hvServer $GlobalConfig.Global.Hyperv.Hosts.ChildNodes[0].ServerName `
-         -TestParams $TestParams
+         -RootDir $WorkingDirectory -TestParams $TestParams
```

Comment:
Better use $Global:WorkingDirectory to easily identify the use of global variable.
And because, this is global variable, you don't need to parameterize it to Main function.